libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -79,6 +79,16 @@ def __init__(self, **kwargs: Any):
             self.model_name, cache_folder=self.cache_folder, **self.model_kwargs
         )
 
+        if "show_progress_bar" in self.encode_kwargs:
```

Comment:
@JonZeolla do we need to update the logic for the two implementations below?

Reply:
@eyurtsev sorry; moved this to draft. Give me some time to test the requested changes locally. I'll @ and set it ready for review when ready